### DISCUSSION TITLE
Import-Module: The specified module "oh-my-posh" was not loaded because no valid module file was found in any module directory.

### DISCUSSION THREAD

**Discussion by noluyorAbi** [JanDeDobbeleer/oh-my-posh#4809] [Troubleshoot] [answered]
Dear oh-my-posh Community,

I am seeking guidance on completely uninstalling oh-my-posh and eliminating the persistent error messages I receive upon opening PowerShell. Despite using the command `winget uninstall JanDeDobbeleer.OhMyPosh -s winget` in an attempt to remove oh-my-posh, I am still confronted with errors related to the module during PowerShell startup:

```
Import-Module: The specified module "oh-my-posh" was not loaded because no valid module file was found in any module directory.
At C:\Users\alper\OneDrive\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1:2 char:1
+ Import-Module oh-my-posh
    + CategoryInfo : ResourceUnavailable: (oh-my-posh:String) [Import-Module], FileNotFoundException
    + FullyQualifiedErrorId : Modules_ModuleNotFound,Microsoft.PowerShell.Commands.ImportModuleCommand

Set-PoshPrompt: The naming "Set-PoshPrompt" was not recognized as the name of a cmdlet, function, script file, or executable program.
At C:\Users\alper\OneDrive\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1:3 char:1
+ Set-PoshPrompt Paradox
    + CategoryInfo : ObjectNotFound: (Set-PoshPrompt:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

I wish to entirely remove oh-my-posh from my system and stop these errors from appearing. If there are steps beyond the uninstallation command I used, or specific configurations I should be aware of, please advise. 

Your expert advice on how to proceed would be invaluable to me. Thank you for your support!

**Comment by JanDeDobbeleer** [comment, marked as answer]
You still have an oh-my-posh statement in your $PROFILE. Open that and remove it.

**Reply by noluyorAbi** [reply]
Thank you very much. Sorry for the disturbance :)
